libr/fs/squashfs/unsquashfs.c

```diff
@@ -1898,14 +1905,14 @@ void initialise_threads(int fragment_buffer_size, int data_buffer_size) {
 			"\n");
 
 	if (processors == -1) {
-#if __linux__
+#if __GLIBC__ == 2 && __GLIBC_MINOR__ <= 30
 		processors = sysconf(_SC_NPROCESSORS_ONLN);
```

Comment:
I think its safe to remove this code. R2 doesnt needs to get the number of processors of anything. Its just a filesystem. Can u remove it? Nice catch!

Reply:
For sure!